x-pack/legacy/plugins/infra/public/routes.tsx

```diff
@@ -37,15 +37,22 @@ const PageRouterComponent: React.SFC<RouterProps> = ({ history, uiCapabilities }
         {uiCapabilities.infrastructure.show && (
           <Redirect from="/home" exact={true} to="/infrastructure/inventory" />
         )}
+        {uiCapabilities.infrastructure.show && (
+          <Route path="/infrastructure/metrics/:type/:node" component={MetricDetail} />
+        )}
+        {uiCapabilities.infrastructure.show && (
+          <Redirect
+            from="/metrics/:type/:node"
+            exact={true}
+            to="/infrastructure/metrics/:type/:node"
```

Comment:
Unfortunately this loses the query params such as the time and thereby breaks previous bookmarks. Maybe we can preserve that via the trick described in https://github.com/ReactTraining/react-router/issues/5818#issuecomment-379212014?

The other redirects in this file seem to suffer from the same problem. :see_no_evil: